Mathlib/Analysis/InnerProductSpace/Adjoint.lean

```diff
@@ -288,20 +288,41 @@ theorem _root_.LinearMap.IsSymmetric.isSelfAdjoint {A : E →L[𝕜] E}
     (hA : (A : E →ₗ[𝕜] E).IsSymmetric) : IsSelfAdjoint A := by
   rwa [← ContinuousLinearMap.isSelfAdjoint_iff_isSymmetric] at hA
 
+end IsSelfAdjoint
+
 /-- The orthogonal projection is self-adjoint. -/
-theorem _root_.orthogonalProjection_isSelfAdjoint (U : Submodule 𝕜 E) [CompleteSpace U] :
-    IsSelfAdjoint (U.subtypeL ∘L U.orthogonalProjection) :=
+@[simp]
+theorem _root_.isSelfAdjoint_starProjection [CompleteSpace E]
+    (U : Submodule 𝕜 E) [U.HasOrthogonalProjection] :
```

Comment:
because if you move the end of the section (which is why this originally had `_root_.`) then `[CompleteSpace E]` is in scope again.
```suggestion
theorem _root_.isSelfAdjoint_starProjection (U : Submodule 𝕜 E) [U.HasOrthogonalProjection] :
```